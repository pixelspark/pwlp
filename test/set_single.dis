0000.	11	PUSHB	[01]
0002.	11	PUSHB	[02]
0004.	11	PUSHB	[ff]
0006.	85	BINARY	AND
0007.	11	PUSHB	[03]
0009.	11	PUSHB	[ff]
0011.	85	BINARY	AND
0012.	74	UNARY	SHL8
0013.	86	BINARY	OR
0014.	11	PUSHB	[04]
0016.	11	PUSHB	[ff]
0018.	85	BINARY	AND
0019.	74	UNARY	SHL8
0020.	74	UNARY	SHL8
0021.	86	BINARY	OR
0022.	e3	USER	set_pixel
0023.	01	POP	1
0024.	e4	USER	blit

