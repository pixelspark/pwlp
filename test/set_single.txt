set_pixel(1, 2, 3, 4);
blit;