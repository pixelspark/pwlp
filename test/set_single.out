000000 020304 000000 000000 000000 000000 000000 000000 000000 000000 
